lib/anyway/loaders/yaml.rb

```diff
@@ -30,9 +30,9 @@ def parse_yml(path)
         # empty. When this occurs, we return an empty hash instead, to match
         # the interface when no config file is present.
         if defined?(ERB)
-          ::YAML.load(ERB.new(File.read(path)).result) || {} # rubocop:disable Security/YAMLLoad
+          ::YAML.load(ERB.new(File.read(path)).result, aliases: true) || {} # rubocop:disable Security/YAMLLoad
```

Comment:
Looks like we need to fallback to the old version for older Rubies. See how Rails does this:https://github.com/rails/rails/commit/179d0a1f474ada02e0030ac3bd062fc653765dbe